### ISSUE TITLE
Unify carry capacity

### ISSUE THREAD

**Issue by Lupus590** [34th-Para-Recon-Brimstone-Company/34th-PRC-Aux-Mod#253]
-ODST uniforms are too small, make them the same size as the marine uniforms.
-Maybe shrink the ODST vests to match the marine ones?
- Maybe set ourselves all of the sizes?
- Add extra to SPI vests to compensate for the small exosuit.
- Spartan stuff can be bigger, they are being trusted to be spartans. They can be trusted to have extra carry capacity too.

- Ballistic protection values too? Maybe this should be its own ticket.

**Comment by Lupus590** [comment]
The shoulderless BDU having less capacity may be due to a bug in MA. 

![image](https://github.com/user-attachments/assets/929e6ec4-4dfe-47e3-ad09-d536dba27313)

**Comment by Lupus590** [comment]
```c++

	class ItemInfo: UniformItem
	{
		containerClass="Supply150";
	};
```

**Comment by Lupus590** [comment]
Blocked, waiting on staff feedback
